src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -192,7 +192,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
       <TargetFramework>$(_TFM)</TargetFramework>
       <RuntimeIdentifier>$(RuntimeIdentifier)</RuntimeIdentifier>
-      <RuntimeFrameworkVersion Condition="'$(FX_Version)' != ''">$(FX_Version)</RuntimeFrameworkVersion>
+      <RuntimeFrameworkVersion Condition="'$(RuntimeFrameworkVersion)' != ''">$(RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
```

Comment:
I don't see how this line does anything. It either does nothing when RuntimeFrameworkVErsion is empty or sets RuntimeFrameworkVersion to itself.